docker-compose.yml

```diff
@@ -42,7 +42,14 @@ services:
       context: ../funding-service-design-post-award-data-store
       args:
         REQUIREMENTS: requirements-dev.txt
-    command: celery -A app.celery_app worker --loglevel INFO
+    command: >
+      /bin/sh -c "
+      if [ \"$CELERY_AUTO_REFRESH\" = \"true\" ]; then
+          watchmedo auto-restart --patterns='*.py' --recursive -- celery -A app.celery_app worker --loglevel INFO;
+      else
+          celery -A app.celery_app worker --loglevel INFO;
+      fi
+      "
```

Comment:
do we need the env var? Our flask app autoreloads by itself - should the celery worker just always auto reload too? Not sure it needs to be configurable

Reply:
I thought it's a useless thing 100% of the times, when we don't change the `async_download` function.

I set it to refresh on any `*.py` change.. so I thought having it disabled by default to save a dolphin or two over time.